### PR TITLE
implement ALEX specific DAO features

### PR DESCRIPTION
**Pull request by fiftyeightandeight**

### CODE REVIEW

File: Clarinet.toml [hunk 1]
```diff
@@ -1,47 +1,32 @@
 [project]
 name = "executor-dao"
 requirements = []
-costs_version = 2
+costs_version = 1
```
Comment (MarvinJanssen) [inline Clarinet.toml:4]: Why was it changed to `1`?

File: contracts/extensions/age003-emergency-execute.clar [hunk 2]
```diff
@@ -55,6 +56,7 @@
 (define-public (set-signals-required (new-requirement uint))
 	(begin
 		(try! (is-dao-or-extension))
+		(asserts! (> new-requirement u0) err-invalid-signals)
```
Comment (MarvinJanssen) [inline contracts/extensions/age003-emergency-execute.clar:59]: I'm not sure if it makes a difference, both `u0` and `u1` would trigger an execution immediately when `executive-action` is called.

File: contracts/proposals/agp002-launchpad-alex.clar [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+(define-constant ONE_8 (pow u10 u8))
+
+(define-public (execute (sender principal))
+	(begin
+		(try! (contract-call? .age000-governance-token mint-fixed (* amount-per-ticket tickets ONE_8) .executor-dao))
+		(try! (contract-call? .alex-launchpad create-pool .age000-governance-token .lottery-ido-alex fee-to-address amount-per-ticket wstx-per-ticket-in-fixed registration-start registration-end claim-end activation-threshold))
+		(contract-call? .alex-launchpad add-to-position .age000-governance-token tickets)
+	)
+)
```
Comment (MarvinJanssen) [inline contracts/proposals/agp002-launchpad-alex.clar:20]: Nice 👍

File: settings/Development.toml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+[network]
```
Comment (MarvinJanssen) [inline settings/Development.toml:1]: `Development.toml` is deprecated in favour of `Devnet.toml`.

**Review by MarvinJanssen — APPROVED** [review]
Looks good. Small comment for `Development.toml`.

File: Clarinet.toml (hunk 1 above)
Comment (fiftyeightandeight) [reply Clarinet.toml:4]: I was testing what the difference is. Will revert to `cost_version = 2`.

File: contracts/extensions/age003-emergency-execute.clar (hunk 2 above)
Comment (fiftyeightandeight) [reply contracts/extensions/age003-emergency-execute.clar:59]: True, let's remove.

**Pull request merged**